articles/templating-your-static-site/index.md

```diff
@@ -0,0 +1,188 @@
+
+Create a folder in the root of your website called `_includes`. This is where Eleventy looks for templates.
+
+**Tip:** A naming convention you may wish to use is to add an `_` to your template files so `_head.ejs`.
+
+In your `_includes` folder, add a file called `_head.ejs` and copy and paste the content of your website's head section. Don't worry if the content differs between pages such as the title tag. We will cover how to use EJS variables with Eleventy in the Adding More Front-Matter section **Anchor Link** so such values can be unique for each page.
+
+Add new partial files for other repeated elements such as the header and footer.
+
+### Creating Your First Layout (Template) File
+
+With the partials created, you can now create your first layout file (template.)
+
+The first layout (often named base layout) will form the basic template of all your pages and will contain the EJS partials (head, header and footer) that you've just created.
+
+**Tip:** Another naming convention that you may want to use is to add -layout at the end of your layout files. This helps to differentiate them from your partials.
+
+In the `_includes` folder, create a file called `_base-layout.ejs` file and add the following to it:
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+    <% include _head.ejs %>
+</head>
+<body>
+    <% include _header.ejs %>
+    <main>
+        <%- content %>
+    </main>
+    <% include _footer.ejs %>
+</body>
+</html>
+```
+If you're new to EJS, `<% include` is the code (syntax) for including a partial file and `<%-` is for including an EJS variable. Content is a variable Eleventy uses to add the body content of HTML, Markdown and other input files.
+
+Congratulations, you've just created your first Eleventy layout file.
+
+###  Rendering Your First Layout File
+
+We have our basic layout template but we need to tell Eleventy where to use it. This is where front-matter comes in.
+
+Front-Matter allows you to define variables you can use in your layout templates both built-in (from Eleventy) and those you've created yourself. By default, Eleventy uses YAML for front-matter though you can change it to JSON or even JavaScript if you prefer.
+
+The most basic front-matter variable, layout will instruct Eleventy which layout file to use to render a page.
+
+On one of the HTML pages of your site, remove the code you included in EJS partials (head, header and footer if you followed the tutorial.)
+
+Then, add the following layout front-matter at the beginning:
+
+```yaml
+---
+layout: _base-layout.ejs
+---
+```
+The three dashes define the start and the end of the front-matter.
+
+Run `npx @11ty/eleventy --serve` in the terminal to start a local web-server for your site that will reload on changes.
```

Comment:
The explanation of this command is vague. A little bit more is needed to make it clear to the reader.

Reply:
Added a more detailed explanation in f83d750. Included details and link about Browsersync.